.github/workflows/npm-check-updates.yml

```
@@ -0,0 +1,133 @@
+        id: verify-changed-files
+
+      - name: Config git
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
+        run: |
+          git config --local user.email "150967461+gstraccini[bot]@users.noreply.github.com"
+          git config --local user.name "gstraccini[bot]"
+          git config --global --add --bool push.autoSetupRemote true
+      
+      - name: Commit files
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:38: Double quote to prevent globbing and word splitting

(shellcheck)